Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/OS - Power and Sleep Actions.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "When the power button is pressed, put the machine to sleep. When the lid is closed on a laptop while plugged in to a power source, the device will not go to sleep.  When the lid for a laptop is closed, and the laptop is not connected to a power source, put the laptop to sleep.\"",
+  "description": "What is it: When the power button is pressed, put the machine to sleep. When the lid is closed on a laptop while plugged in to a power source, the device will not go to sleep.  When the lid for a laptop is closed, and the laptop is not connected to a power source, put the laptop to sleep.",
```

Comment:
**What does this do?**
When the power button is pressed, the machine goes to sleep. When the laptop lid is closed and the laptop is not plugged in to a power source, the device will go to sleep. When the laptop lid is closed while plugged in to a power source, the device will not go to sleep. 

**Why should you use this?**
If you want users to save battery.

**What is the end-user impact?**
Users' laptops will go to sleep when they press the power button and when the laptop lid is closed and not plugged in to a power source. Users' laptops will not go to sleep when the laptop lid is closed and plugged in to a power source.

**Learn more**
N/A

@metamenos should this be in the baseline...? Maybe there's some reason for use that I'm missing